learn_script.py

```diff
@@ -80,7 +95,7 @@ def run_command(content, free=False, auto=False):
         )
         process.stdin.write(user_input.encode("utf-8") + b"\n")
         for line in process.stdout.readlines():
-            ansiprint(output + line.decode("utf-8"), end="")
+            ansiprint(output + line.decode("utf-8"))
```

Comment:
process.stdout line will include a EoL. without each line will have two new lines.

```suggestion
             ansiprint(output + line.decode("utf-8"), end="")
```